src/test/resources/fixtures/hmh/hmh-interacted/development/caliperEventAssessementStarted.json

```diff
@@ -0,0 +1,185 @@
+            "id": "urn:uuid:13ca9c97-80f8-4136-ac84-532799feda47",
+            "type": "AssessmentEvent",
+            "actor": {
+                "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+                "id": "urn:uuid:ff7c53fd-9024-4d2d-8503-aa3da2cbc73e",
+                "type": "Person",
+                "name": null,
+                "description": null,
+                "dateCreated": null,
+                "dateModified": null,
+                "extensions": {}
+            },
+            "action": "§ted",
+            "object": {
+                "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+                "id": "data-uuid-e767f484ee4c4718a1cb05671100deb2",
+                "type": "Assessment",
+                "name": "CPL v2 Flip Cards",
+                "description": "CPL Flip Cards",
+                "keywords": [],
+                "isPartOf": null,
+                "extensions": {
+                    "assignmentId": null,
+                    "resourceId": null,
+                    "contentRootId": "080c58111f020b9418f824970ffd7bb5392",
+                    "contentRootType": "pubId",
+                    "contentVersionId": 0,
+                    "userRole": "student",
+                    "contentItemId": "d5763388b980473ebbfce7876b23e345"
+                },
+                "dateCreated": "2019-06-10T15:50:19.997Z",
+                "dateModified": "2019-06-10T15:50:19.997Z",
+                "datePublished": "2019-06-10T15:50:19.997Z",
+                "version": "1.0",
+                "dateToActivate": "2019-06-10T15:50:19.997Z",
+                "dateToShow": "2019-06-10T15:50:19.997Z",
+                "dateToStartOn": "2019-06-10T15:50:19.997Z",
+                "dateToSubmit": "2019-06-10T15:50:19.997Z",
+                "maxAttempts": 0,
+                "maxSubmits": 0,
+                "maxScore": 0
+            },
+            "eventTime": "2019-06-10T14:50:36.274Z",
+            "generated": {},
+            "target": {
+                "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+                "id": "http://rce.localhost/content/book/OPS/s9ml/cards/el_uat_content_samp_011.xhtml",
+                "type": "Frame",
+                "name": "Flip Cards",
+                "description": null,
+                "dateCreated": null,
+                "dateModified": null,
+                "extensions": {
+                    "programId": null,
+                    "programName": null,
+                    "contentRootId": "080c58111f020b9418f824970ffd7bb5392",
+                    "contentRootType": "pubId",
+                    "contentVersionId": 0,
+                    "userRole": "student",
+                    "contentItemId": "d5763388b980473ebbfce7876b23e345"
+                },
+                "mediaType": null,
+                "creators": [],
+                "keywords": [],
+                "learningObjectives": [],
```

Comment:
any things that empty or null you can just leave it out of the event unless you are calling out that they are meant to empty as opposed to null?

Reply:
fixed